apps/api/src/app.module.ts

```
@@ -27,8 +27,20 @@ import { ContactModule } from './modules/contact/contact.module';
         QIITA_USER_ID: Joi.string().trim().min(1).required(),
         RECAPTCHA_SECRET_KEY: Joi.string().allow('').default(''),
         RESEND_API_KEY: Joi.string().allow('').default(''),
-        RESEND_FROM: Joi.string().email().default('noreply@example.com'),
-        RESEND_TO: Joi.string().email().default('admin@example.com'),
+        RESEND_FROM: Joi.string()
+          .default('noreply@example.com')
+          .when('RESEND_API_KEY', {
+            is: Joi.string().min(1),
+            then: Joi.string().min(1).required(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)